configure

```diff
@@ -9814,7 +9814,7 @@ then
 		# to ensure that tests don't crash
 		# Note: This matches the value of THREAD_STACK_SIZE in
```

Comment:
As per this comment does this need to be kept up to date with:

https://github.com/python/cpython/blob/11749e2dc20ad6a76e9a39e948853e89b2b4bbed/Python/thread_pthread.h#L35-L46

Reply:
That's already done, see the invocation of AC_DEFINE_UNQUOTED below (line 2849 as I write this comment).